doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,94 +6,131 @@ Cite Test Guide
 A step by step guide to the GeoServer Compliance Interoperability Test Engine (CITE).
 
 .. contents::
-   :depth: 2
+~~~~~~~~~~~~~
 
-Check out CITE tools
---------------------
+Check out CITE suite tools
+--------------------------
 
-The CITE tools are available at https://github.com/jdeolive/geoserver-cite-tools. The README file 
-contains the most update documentation of how to checkout and build the tools. The quick version is::
+.. note:: The CITE tools are available at `Open Geospatial Consortium`_.
+.. _Open Geospatial Consortium: https://github.com/opengeospatial
+
+Requirements:
+-------------
+
+- `GeoServer <https://github.com/geoserver/geoserver>`_.
+
+- :ref:`Teamengine Web Application<Teamengine Web Application>`
+
+.. note::
+
+   To build every suite test you need to do the next steps:
+
+  .. code:: shell
+
+      git clone https://github.com/opengeospatial/ets-wcs10.git
+      cd ets-wcs10.git
+      mvn install
 
-  git clone git://github.com/jdeolive/geoserver-cite-tools.git
-  cd geoserver-cite-tools
-  git submodule update --init
-  mvn clean install
-  
 Run WFS 1.0 tests
 -----------------
 
-.. note::
+.. important::
 
    Running WFS 1.0 tests require PostGIS to be installed on the system.
 
-#. Create a PostGIS user named "cite"::
+Requirements:
+~~~~~~~~~~~~~
+- Posgresql
+- PostGIS
 
-     createuser cite
+#. Create a PostGIS user named "cite":
 
-#. Create a PostGIS databased named "cite", owned by the "cite" user::
+   .. code:: SQL
 
-     createdb -T template_postgis -U cite cite
+      CREATEUSER cite
+
+#. Create a PostGIS databased named "cite", owned by the "cite" user:
+
+   .. code:: SQL
 
-#. Change directory to the ``citewfs-1.0`` data directory and execute the script
-   ``cite_data.sql``::
+      CREATEDB -T template_postgis -U cite cite
```

Comment:
This would not work on my machine, the ``template_postgis`` template db is legacy, the normal way to enable a database to use postgis is:

``psql mydb -c "create extension postgis"``